app/brave_main_delegate_browsertest.cc

```diff
@@ -44,7 +44,6 @@ IN_PROC_BROWSER_TEST_F(BraveMainDelegateBrowserTest, DisabledFeatures) {
       &autofill::features::kAutofillServerCommunication,
       &features::kAllowPopupsDuringPageUnload,
       &features::kAudioServiceOutOfProcess,
-      &features::kDefaultEnableOopRasterization,
```

Comment:
My bad on this one! I must have missed with https://github.com/brave/brave-core/pull/4318 (original PR https://github.com/brave/brave-core/pull/3793 didn't add a test). I'm curious why it hasn't been failing on C79? (ex: master)